### ISSUE TITLE
IBM part audit is still failing

### ISSUE THREAD

**Issue by jmartin-sul** [sul-dlss/preservation_catalog#1337]
error seems to be the same as the one in #1307 (fixed in #1310 for audit of replicated parts on AWS):

```
Worker
    preservation-catalog-stage-02.stanford.edu:29862 on part_audit_ibm_us_south at about a minute ago

Class
    PartReplicationAuditJob (via ActiveJob) 

Arguments

    ---
    job_class: PartReplicationAuditJob
    job_id: cebfa9a1-4554-4fae-a18d-831355a5d7b7
    provider_job_id: 
    queue_name: part_audit_ibm_us_south
    priority: 
    arguments:
    - _aj_globalid: gid://preservation-catalog/CompleteMoab/9181
    - _aj_globalid: gid://preservation-catalog/ZipEndpoint/3
    executions: 0
    exception_executions: {}
    locale: en
    timezone: UTC
    enqueued_at: '2020-01-23T22:40:25Z'

Exception
    NoMethodError
Error
    undefined method `[]' for nil:NilClass
```

**Comment by jmartin-sul** [comment]
ok computer is complaining about the IBM part audit failure queue depth:
`feature-part_audit_ibm_us_south_failed-queue-threshold: FAILED part_audit_ibm_us_south_failed is 196940 over threshold! (196960)`

and indeed, resque dashboard indicates the same size for that failure queue.  further, if a failed job is re-queued, it fails immediately with the same error.  audit for the two AWS endpoints does appear to be fixed, though (e.g., if i call `#audit_moab_version_replication!` on a random `CompleteMoab` on stage, i see the error described in this issue added to the IBM part audit failure queue, but the AWS part audit jobs seem to complete without a problem).

**Comment by jmartin-sul** [comment]
manually reproduced on rails console on stage yesterday.

manually tested a fix along the lines of #1310 on stage yesterday, and that seemed to allow the IBM part audit class to call `#bucket` successfully.

will deploy a branch w/ the fix to stage and test more realistically/less manually (i.e. by re-queuing a job from resque dashboard).
